### DISCUSSION TITLE
problem with Debian 10

### DISCUSSION THREAD

**Discussion by erivansilva** [Chia-Network/chia-blockchain#6001] [Support]
Good Morning.
I am trying to connect my wallet to the Linux / Debian 10 operating system, but just keep trying and never count.
Thank you in advance.
------------------------------------------------------------------------------------------------------------------------------------------------------------
Bom dia.

Estou tentando conctar a minha carteira no sistema operacinal linux/Debian 10 porém só fica tentando e nunca conctar. 

Grato desde já.

![Captura de tela de 2021-05-25 09-51-22](https://user-images.githubusercontent.com/69176863/119501158-fcb0cf80-bd3e-11eb-910c-e341ad62188b.png)

**Comment by erivansilva** [comment]
**running through the terminal I had the following error:**

_chia-blockchain 
/usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon
true
Error loading config - using defaults
startUrl file:///usr/lib/chia-blockchain/resources/app.asar/build/renderer/index.html
/usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon
true
/usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon
true
Running python executable: 
child process success
(node:8646) electron: The default of contextIsolation is deprecated and will be changing from false to true in a future release of Electron.  See https://github.com/electron/electron/issues/23506 for more information
No cert
Have cert
stderr: Traceback (most recent call last):
  File "chia/daemon/server.py", line 995, in <module>
  File "chia/daemon/server.py", line 991, in main
  File "chia/daemon/server.py", line 985, in run_daemon
  File "asyncio/base_events.py", line 616, in run_until_complete
  File "chia/daemon/server.py", line 977, in async_run_daemon
  File "chia/daemon/server.py", line 143, in start
  File "websockets/server.py", line 965, in __await_impl__
  File "asyncio/base_events.py", line 1463, in create_server
OSError: [Errno 99] error while attempting to bind on address ('::1', 55400, 0, 0): cannot assign requested address
[8694] Failed to execute script server
closing code: 255_

**Reply by 7by24** [reply]
ipv6 issue. you should change the localhost to 127.0.0.1 in config.yaml

**Reply by erivansilva** [reply]
> ipv6 issue. you should change the localhost to 127.0.0.1 in config.yaml

it worked, thanks for the tip.

_funcionou, valeu pela dica._

**Comment by BrandtH22** [comment]
@erivansilva , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
